UnicornRender/video/include/unicorn/video/Primitives.hpp

```diff
@@ -57,6 +57,7 @@ class Primitives
     * Supported formats:
     * - gltf 2.0 (without binary glb)
     * - obj
+    * - fbx (without embedded textures)
```

Comment:
how is this related to the world right-handedness?

Reply:
FBX was added for visual transformation tests, because gltf has bug and obj bake all transformation in vertices